colortv/docs/androidTVSDK.md

```diff
@@ -194,9 +194,55 @@ colorTvRecommendationsController.showRecommendationCenter(ColorTvPlacements.VIDE
 
 Invoking this method will show Recommendation Center for the placement you pass. You need to call the `load` method for a given placement before invoking `showRecommendationCenter` in order to load recommendations related data. Also make sure you got the `onLoaded` callback first, otherwise the Recommendation Center won't be displayed.
 
+```java
+ColorTvRecommendationCenterFragment recommendationCenterFragment = colorTvRecommendationsController.getRecommendationCenterFragment(ColorTvPlacements.MAIN_MENU);
+```
+
+Invoking this method will return `ColorTvRecommendationCenterFragment` class (which inherits `android.support.v4.app.Fragment`) for the placement you pass. When you get fragment you can put it in any place you want by using `FragmentManager`. It gives a lot of flexibility and capabilities to make Recommendation Center a part of your layout (i.e. you can add it between videos list, use it in vertical/horizontal layout).
+
+**ColorTvRecommendationCenterFragment** provides few methods that allows you to make it work as you need.
+
+```java
+recommendationCenterFragment.setRequestFocusOnStart(boolean enabled)
+```
+
+This method is used to request focus on first element when Recommendation Center will completely load. Should be used before replacing/adding fragment. To make it working properly you should avoid set this method to true in more than one fragment on the screen.
+
+```java
+recommendationCenterFragment.setConfig(ColorTvContentRecommendationConfig config);
+```
+
+This method allows you to pass separate config for each fragment. We added to `ColorTvContentRecommendationConfig` new method `getCopy()` which copies current config to new instance. If you need similar config it will be easier, faster and safer to copy config and change only differing parameter.
+
+To make Recommendation Center working properly you have to pass below methods invocations in activity's 'dispatchKeyEvent' and `dispatchTouchEvent` method which operates on received `ColorTvRecommendationCenterFragment`.
```

Comment:
To make Recommendation Center work properly, you have to invoke the following methods in activity's

dispatchKeyEvent is in the wrong type of quotation marks